### ISSUE TITLE
Components annotations retention policy

### ISSUE THREAD

**Issue by pavi2410** [mit-cml/appinventor-sources#1744]
https://github.com/mit-cml/appinventor-sources/blob/77eccf1c640ca7bef6a25f5f4e3ec1115e0ed18b/appinventor/components/src/com/google/appinventor/components/annotations/androidmanifest/ActivityElement.java#L31

Should the annotations in `androidmanifest` directory be retained only at `SOURCE` level? I think that these annotations are not used at runtime (?)

**Comment by ewpatton** [comment]
@pavi2410 I don't believe we currently use the annotation on the device, so I expect that you are correct here. Have you tried switching it to `SOURCE`?

**Comment by pavi2410** [comment]
@ewpatton Is building the companion alone and run it enough to test this?

**Comment by ewpatton** [comment]
I believe so, but we should audit everything that gets compiled into AndroidRuntime.jar and runtime.scm to see if anything still references the annotations after compilation.
